conda_smithy/configure_feedstock.py

```diff
@@ -1653,13 +1653,13 @@ def _github_actions_specific_setup(jinja_env, forge_config, forge_dir, platform)
     runs_on = {
         "osx-64": {
             "os": "macos",
-            "hosted_labels": ("macos-13",),
+            "hosted_labels": ("macos-15",),
```

Comment:
Using this on Github actions will switch osx-64 builds from native to cross. Please revert this change. The correct solution is to either
1. Keep using macos-13
2. Use macos-15, but provision a osx-64 environment for building.

Reply:
done!